examples/ttClosedTemplate.json

```diff
@@ -0,0 +1,82 @@
+{
+  "device_id": "{{deviceId}}",
+  "protocol": "TT_CLOSED",
+  "entity_name": "{{deviceName}}",
+  "entity_type": "{{deviceType}}",
+  "attributes": [
```

Comment:
Not necessarily. TT_OPEN, as its name suggest, is open to include modules for different generic values. For TT_OPEN devices, the template must be tailored to the specific case. The same would be true for a "generic" TT Closed, i.e.: a TTClosed with a different set of modules as the ones we use. This template, anyway, has been uploaded as an example of how to create templates for the bulk loading, not to be a standard way of provisioning (see the "/examples" folder).